google-http-client/src/main/java/com/google/api/client/util/SslUtils.java

```diff
@@ -109,6 +109,36 @@ public static SSLContext initSslContext(
     return sslContext;
   }
 
+  /**
+   * Initializes the SSL context to the trust managers supplied by the trust manager factory for the
+   * given trust store, and to the key managers supplied by the key manager factory for the given
+   * key store.
+   *
+   * @param sslContext SSL context (for example {@link SSLContext#getInstance})
+   * @param trustStore key store for certificates to trust (for example {@link
+   *     SecurityUtils#getJavaKeyStore()})
+   * @param trustManagerFactory trust manager factory (for example {@link
+   *     #getPkixTrustManagerFactory()})
+   * @param keyStore key store for client certificate and key
+   * @param keystorePassword password for keyStore parameter
+   * @param keyManagerFactory key manager factory (for example {@link
+   *     #getDefaultKeyManagerFactory()})
+   */
+  public static SSLContext initSslContext(
+      SSLContext sslContext,
+      KeyStore trustStore,
+      TrustManagerFactory trustManagerFactory,
+      KeyStore keyStore,
+      char[] keystorePassword,
```

Comment:
`char[]` is not a friendly Java construct. Consider `String` and call `toCharArray()`

Reply:
done